### PR TITLE
Fix debug logging doesn't show module name

### LINKED ISSUE #1988: init with debug logging doesn't show module name for available versions
### OpenTofu Version

```shell
OpenTofu v1.8.2
on darwin_arm64
+ provider registry.terraform.io/hashicorp/google v6.2.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.32.0
+ provider registry.terraform.io/hashicorp/random v3.6.3
```

### OpenTofu Configuration Files

```hcl
provider "google" {
  project = "xxxx"
  region  = "us-central1"
  zone    = "us-central1-f"
}

terraform {
  required_providers {
    google = {
      source  = "hashicorp/google"
      version = "6.2.0"
    }
  }
}

module "gke" {
  source                     = "terraform-google-modules/kubernetes-engine/google//modules/private-cluster"
  version                    = "33.0.2"   // This will be available at some point, but could put any non-existent version in here.
  project_id                 = "xxxx"
  name                       = "test-cluster"
  service_account_name       = "foobarbaz"
  grant_registry_access      = true
  kubernetes_version         = "1.30.3-gke.1969000"
  release_channel            = "UNSPECIFIED"
  region                     = "us-central1"
  zones                      = ["us-central1-f"]
  network                    = "default"
  subnetwork                 = "us-central1"
  horizontal_pod_autoscaling = true
  enable_private_nodes       = true
  dns_cache                  = true
  ip_range_pods              = "foo-pods"
  ip_range_services          = "foo-services"
}
```

### Debug Output

https://gist.github.com/wyardley/0cccc2a773d218db415e3c477b1d77a2

### Expected Behavior

The debug output should show the name of the module for each version found, as it does with Terraform 1.5.x and 1.9.x (the versions I checked) e.g.:

```
2024-09-13T11:40:14.782-0700 [DEBUG] found available version "29.0.0" for terraform-google-modules/kubernetes-engine/google
```

### Actual Behavior

```
2024-09-13T11:38:03.399-0700 [DEBUG] found available version "29.0.0" for
```

### Steps to Reproduce

1. `tofu init -upgrade`

### Additional Context

From what I remember, the equivalent code for providers (vs. modules) does work properly in OpenTofu, but I didn't test extensively.

### References

_No response_

### PR DESCRIPTION
**Pull request by arunim2405**
https://gist.github.com/arunim2405/93d536ad37ad481a49c8c68a3dcc5337

Resolves #1988

## Target Release

1.9.0

## Checklist

- [x] I have read the [contribution guide](https://github.com/opentofu/opentofu/blob/main/CONTRIBUTING.md).
- [x] I have not used an AI coding assistant to create this PR.
- [x] I have written all code in this PR myself OR I have marked all code I have not written myself (including modified code, e.g. copied from other places and then modified) with a comment indicating where it came from.
- [x] I (and other contributors to this PR) have not looked at the Terraform source code while implementing this PR.

### Go checklist

 

- [x] I have run golangci-lint on my change and receive no errors relevant to my code.
- [x] I have run existing tests to ensure my code doesn't break anything.
- [x] I have added tests for all relevant use cases of my code, and those tests are passing.
- [x] I have only exported functions, variables and structs that should be used from other packages.
- [x] I have added meaningful comments to all exported functions, variables, and structs.

### Website/documentation checklist

- [ ] I have locally started the website as [described here](https://github.com/opentofu/opentofu/blob/main/website/README.md) and checked my changes.

### CODE REVIEW

**Comment by arunim2405** [comment]
`2024-09-21T11:22:52.055+0530 [INFO]  OpenTofu version: 1.9.0 dev
2024-09-21T11:22:52.056+0530 [DEBUG] using github.com/hashicorp/go-tfe v1.36.0
2024-09-21T11:22:52.056+0530 [DEBUG] using github.com/opentofu/hcl/v2 v2.0.0-20240814143621-8048794c5c52
2024-09-21T11:22:52.056+0530 [DEBUG] using github.com/hashicorp/terraform-svchost v0.1.1
2024-09-21T11:22:52.056+0530 [DEBUG] using github.com/zclconf/go-cty v1.14.4
2024-09-21T11:22:52.056+0530 [INFO]  Go runtime version: go1.22.3
2024-09-21T11:22:52.056+0530 [INFO]  CLI args: []string{"./tofu", "init", "-upgrade"}
2024-09-21T11:22:52.056+0530 [DEBUG] Attempting to open CLI config file: /Users/arunimchaudhary/.tofurc
2024-09-21T11:22:52.056+0530 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2024-09-21T11:22:52.056+0530 [DEBUG] ignoring non-existing provider search directory terraform.d/plugins
2024-09-21T11:22:52.056+0530 [DEBUG] ignoring non-existing provider search directory /Users/arunimchaudhary/.terraform.d/plugins
2024-09-21T11:22:52.056+0530 [DEBUG] ignoring non-existing provider search directory /Users/arunimchaudhary/Library/Application Support/io.terraform/plugins
2024-09-21T11:22:52.056+0530 [DEBUG] ignoring non-existing provider search directory /Library/Application Support/io.terraform/plugins
2024-09-21T11:22:52.057+0530 [DEBUG] Found the config directory: /Users/arunimchaudhary/.terraform.d
2024-09-21T11:22:52.057+0530 [INFO]  CLI command args: []string{"init", "-upgrade"}

Initializing the backend...
2024-09-21T11:22:52.066+0530 [DEBUG] New state was assigned lineage "0dfe07de-22be-1e22-3484-e94d87765ff8"
2024-09-21T11:22:52.066+0530 [DEBUG] checking for provisioner in "."
2024-09-21T11:22:52.066+0530 [DEBUG] checking for provisioner in "/Users/arunimchaudhary/Desktop/projects/opentofu"
Upgrading modules...
2024-09-21T11:22:52.067+0530 [DEBUG] Module installer: begin gke
2024-09-21T11:22:52.177+0530 [DEBUG] gke listing available versions of registry.opentofu.org/terraform-google-modules/kubernetes-engine/google//modules/private-cluster at registry.opentofu.org
2024-09-21T11:22:52.178+0530 [DEBUG] Service discovery for registry.opentofu.org at https://registry.opentofu.org/.well-known/terraform.json
2024-09-21T11:22:52.286+0530 [DEBUG] fetching module versions from "https://registry.opentofu.org/v1/modules/terraform-google-modules/kubernetes-engine/google/versions"
2024-09-21T11:22:52.286+0530 [DEBUG] GET https://registry.opentofu.org/v1/modules/terraform-google-modules/kubernetes-engine/google/versions
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "33.0.4" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "33.0.3" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "33.0.2" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "33.0.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "33.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "32.0.4" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "32.0.3" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "32.0.2" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "32.0.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "32.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "31.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "31.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "30.3.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "30.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "30.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "30.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "29.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "28.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "27.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "26.1.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "26.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "26.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "25.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "24.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "24.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "23.3.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "23.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "23.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "23.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "22.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "22.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "21.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "21.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "21.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "20.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "19.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "18.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "17.3.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "17.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "17.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "17.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "16.1.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "16.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "16.0.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "16.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "15.0.2" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "15.0.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "15.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "14.3.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "14.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "14.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "14.0.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "14.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "13.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "13.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "12.4.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "12.3.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "12.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "12.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "12.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "11.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "11.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "10.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "9.4.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "9.3.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "9.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "9.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "9.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "8.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "8.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "7.3.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "7.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "7.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "7.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "6.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "6.1.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "6.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "6.0.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "6.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "5.1.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "5.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "5.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "5.0.0-rc1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "4.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "4.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "3.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "2.1.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "2.0.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "2.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "1.0.1" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "1.0.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "0.4.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "0.3.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "0.2.0" for terraform-google-modules/kubernetes-engine/google
2024-09-21T11:22:52.325+0530 [DEBUG] found available version "0.1.0" for terraform-google-modules/kubernetes-engine/google
Downloading registry.opentofu.org/terraform-google-modules/kubernetes-engine/google 33.0.2 for gke...
2024-09-21T11:22:52.326+0530 [DEBUG] looking up module location from "https://registry.opentofu.org/v1/modules/terraform-google-modules/kubernetes-engine/google/33.0.2/download"
2024-09-21T11:22:52.326+0530 [DEBUG] GET https://registry.opentofu.org/v1/modules/terraform-google-modules/kubernetes-engine/google/33.0.2/download
2024-09-21T11:22:56.897+0530 [DEBUG] Module installer: gke installed at .terraform/modules/gke/modules/private-cluster
- gke in .terraform/modules/gke/modules/private-cluster

Initializing provider plugins...
- Finding hashicorp/random versions matching ">= 2.1.0"...
2024-09-21T11:22:56.901+0530 [DEBUG] GET https://registry.opentofu.org/v1/providers/hashicorp/random/versions
- Finding hashicorp/google versions matching ">= 5.40.0, 6.2.0, < 7.0.0"...
2024-09-21T11:22:57.217+0530 [DEBUG] GET https://registry.opentofu.org/v1/providers/hashicorp/google/versions
- Finding hashicorp/kubernetes versions matching "~> 2.10"...
2024-09-21T11:22:57.595+0530 [DEBUG] GET https://registry.opentofu.org/v1/providers/hashicorp/kubernetes/versions
2024-09-21T11:22:58.438+0530 [DEBUG] GET https://registry.opentofu.org/v1/providers/hashicorp/random/3.6.3/download/darwin/arm64
2024-09-21T11:22:58.757+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-random/releases/download/v3.6.3/terraform-provider-random_3.6.3_SHA256SUMS
2024-09-21T11:23:00.328+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-random/releases/download/v3.6.3/terraform-provider-random_3.6.3_SHA256SUMS.sig
- Installing hashicorp/random v3.6.3...
2024-09-21T11:23:01.830+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-random/releases/download/v3.6.3/terraform-provider-random_3.6.3_darwin_arm64.zip
2024-09-21T11:23:05.161+0530 [DEBUG] Validating GPG signature of provider package /var/folders/bl/kp28xc551y504lltqmb7h1k00000gn/T/terraform-provider1924065924
2024-09-21T11:23:05.164+0530 [DEBUG] Provider signed by 0C0AF313E5FD9F80 OpenTofu (This key is used to sign opentofu providers) <core@opentofu.org>
- Installed hashicorp/random v3.6.3 (signed, key ID 0C0AF313E5FD9F80)
2024-09-21T11:23:05.337+0530 [DEBUG] GET https://registry.opentofu.org/v1/providers/hashicorp/google/6.2.0/download/darwin/arm64
2024-09-21T11:23:05.912+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-google/releases/download/v6.2.0/terraform-provider-google_6.2.0_SHA256SUMS
2024-09-21T11:23:07.395+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-google/releases/download/v6.2.0/terraform-provider-google_6.2.0_SHA256SUMS.sig
- Installing hashicorp/google v6.2.0...
2024-09-21T11:23:08.760+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-google/releases/download/v6.2.0/terraform-provider-google_6.2.0_darwin_arm64.zip
2024-09-21T11:23:19.246+0530 [DEBUG] Validating GPG signature of provider package /var/folders/bl/kp28xc551y504lltqmb7h1k00000gn/T/terraform-provider375087379
2024-09-21T11:23:19.246+0530 [DEBUG] Provider signed by 0C0AF313E5FD9F80 OpenTofu (This key is used to sign opentofu providers) <core@opentofu.org>
- Installed hashicorp/google v6.2.0 (signed, key ID 0C0AF313E5FD9F80)
2024-09-21T11:23:20.401+0530 [DEBUG] GET https://registry.opentofu.org/v1/providers/hashicorp/kubernetes/2.32.0/download/darwin/arm64
2024-09-21T11:23:20.925+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-kubernetes/releases/download/v2.32.0/terraform-provider-kubernetes_2.32.0_SHA256SUMS
2024-09-21T11:23:22.550+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-kubernetes/releases/download/v2.32.0/terraform-provider-kubernetes_2.32.0_SHA256SUMS.sig
- Installing hashicorp/kubernetes v2.32.0...
2024-09-21T11:23:24.108+0530 [DEBUG] GET https://github.com/opentofu/terraform-provider-kubernetes/releases/download/v2.32.0/terraform-provider-kubernetes_2.32.0_darwin_arm64.zip
2024-09-21T11:23:30.880+0530 [DEBUG] Validating GPG signature of provider package /var/folders/bl/kp28xc551y504lltqmb7h1k00000gn/T/terraform-provider1491535389
2024-09-21T11:23:30.880+0530 [DEBUG] Provider signed by 0C0AF313E5FD9F80 OpenTofu (This key is used to sign opentofu providers) <core@opentofu.org>`

**Comment by arunim2405** [comment]
https://gist.github.com/arunim2405/93d536ad37ad481a49c8c68a3dcc5337

**Comment by arunim2405** [comment]
@Evi1Pumpkin @abstractionfactory @cam72cam

**Review by ollevche — APPROVED** [review]
Thank you for the fix!

File: internal/registry/client.go [hunk 1]
```diff
@@ -168,7 +168,7 @@ func (c *Client) ModuleVersions(ctx context.Context, module *regsrc.Module) (*re
 
 	for _, mod := range versions.Modules {
 		for _, v := range mod.Versions {
-			log.Printf("[DEBUG] found available version %q for %s", v.Version, mod.Source)
+			log.Printf("[DEBUG] found available version %q for %s", v.Version, module.Module())
```
Comment (cam72cam) [inline internal/registry/client.go:171]: Could we keep mod.Source in the log here?

`found available version %q of %q for %s`, version, source, module

Comment (arunim2405) [reply internal/registry/client.go:171]: What if mod.Source is empty? Should we still print `found available version x.x.x of for provider/module/name`

**Review by cam72cam — APPROVED** [review]

**Comment by cam72cam** [comment]
Thanks @arunim2405 !

**Pull request merged**